src/components/DiffView/index.tsx

```diff
@@ -265,6 +284,42 @@ export class DiffViewBase extends React.Component<Props> {
     return <div className={styles.extraMessages}>{messages}</div>;
   };
 
+  renderGutter = ({
+    change,
+    renderDefault,
+    side,
+    wrapInAnchor,
+  }: RenderGutterParams) => {
+    const { enableCommenting, version } = this.props;
+
+    const { isDelete, lineNumber } = change;
+    const changeKey = getChangeKey(change);
+
+    const defaultGutter = wrapInAnchor(renderDefault());
+    let gutter = defaultGutter;
+    if (enableCommenting && !isDelete && lineNumber && side === 'new') {
+      gutter = (
+        <Commentable
+          as="div"
+          id={changeKey}
```

Comment:
You don't have to define an ID here. It's only there in case the wrapped element (in this case `div`) needs an `id` attribute.